simapp/utils_test.go

```diff
@@ -4,39 +4,38 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/cosmos/cosmos-sdk/std"
-
 	"github.com/stretchr/testify/require"
-	tmkv "github.com/tendermint/tendermint/libs/kv"
 
+	"github.com/cosmos/cosmos-sdk/std"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkkv "github.com/cosmos/cosmos-sdk/types/kv"
```

Comment:
nit, I'd just use `kv` if there are no import conflicts

Reply:
there are some type names called kv so it conflicts a bit with them. That is why i went with sdkkv. Its either some are sdkkv and some are kv or all sdkkv. I dont have a preference. which do you prefer?

Reply:
> either some are sdkkv and some are kv

This one is better imo